goerli/.env.example

```diff
@@ -67,7 +67,7 @@ NODE_PUBLIC_IP_CHECK_INTERVAL=10s
 NODE_HOSTNAME=localhost
 
 # TODO: Node Nginx config
-NODE_NGINX_CONF_HOST=./resources/rate-limit-nginx.conf
+NODE_NGINX_CONF_HOST=../resources/rate-limit-nginx.conf
```

Comment:
Should we have separate `EIGENDA_HOME` directory for each network? (i.e. `${EIGENLAYER_HOME}/eigenda/goerli`)

Reply:
My assumption is they won't run two network in same machine ?

Reply:
That's probably true, but i think we should either make that explicit or assume they will do whatever they can

Reply:
yea maybe you are right. quite possible solo stakers use same machine so I can add that.